src/java/generator/MapGenerator.java

```diff
@@ -766,17 +766,24 @@ private void setupResourcePipeline() {
 
     private void setupTexturePipeline() {
         ConcurrentBinaryMask flat = new ConcurrentBinaryMask(slope, .05f, random.nextLong(), "flat").invert();
-        ConcurrentBinaryMask ground = new ConcurrentBinaryMask(land, random.nextLong(), "ground");
+        ConcurrentBinaryMask inland = new ConcurrentBinaryMask(land, random.nextLong(), "inland").invert();
```

Comment:
is this supposed to be inverted? the populator one isnt and the variable name suggests not